static/app/views/releases/components/header.tsx

```diff
@@ -31,17 +31,15 @@ function Header({router, hasV2ReleaseUIEnabled = false}: Props) {
   const tabs = hasV2ReleaseUIEnabled
     ? [
         {
-          key: MONITOR_PATH,
           label: t('Monitor'),
           description: '',
-          path: MONITOR_PATH,
+          path: normalizeUrl(`/organizations/${organization.slug}/releases/`),
```

Comment:
do we need to normalize the urls twice? looks like that's already happening in the map